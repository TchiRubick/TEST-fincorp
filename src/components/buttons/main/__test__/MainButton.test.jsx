import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import MainButton from '../MainButton';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<MainButton>test</MainButton>, div);
});

it('as expected snapshot', () => {
  const tree = renderer.create(<MainButton>test</MainButton>).toJSON();
  expect(tree).toMatchSnapshot();
});
