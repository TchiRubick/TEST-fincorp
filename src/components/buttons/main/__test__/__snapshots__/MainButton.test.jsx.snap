// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1`] = `
<button
  className="comp-button-main"
  type="button"
>
  test
</button>
`;
